applications/asset_tracker_v2/src/events/led_state_event.c

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "led_state_event.h"
+
+
+EVENT_TYPE_DEFINE(led_state_event,
```

Comment:
put those to one line, we will not exceed 100 chars. Its leftover i believe, we should clear this up.